max_test.Rmd

```diff
@@ -59,6 +59,8 @@ defNew
 
 ```
 ```{r}
-assertInteger(as.integer(1))
+test <- c("one", "two", "three")
+
+typeof(assertIntegerOrFactor(var1 = test))
 ```
 
```

Comment:
This test file should probably not be saved in the package, but in another workspace.

Reply:
i just moved it